### PR TITLE
added profile and photo

### PR DESCRIPTION
**Pull request by johnw335**

### CODE REVIEW

**Comment by dblock** [comment]
Please also add a link to README. That would add a commit, please make sure to _squash_ multiple commits. This might be helpful: http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html

File: people/john-williams.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+John Williams
+
+![](photos/john_profile.jpg)
```
Comment (dblock) [inline people/john-williams.md]: I am sure there's another John in the class. Call this john-williams.jpg to match the md file, please.

**Comment by dblock** [comment]
See my other comments please.

**Comment by karnseth** [comment]
Can be closed: John submitted another PR

**Pull request closed without merging**